src/main/java/practice/StreamPractice.java

```diff
@@ -50,7 +70,14 @@ public List<Person> selectMenByAge(List<Person> peopleList, int fromAge, int toA
      */
     public List<Person> getWorkablePeople(int fromAge, int femaleToAge,
                                           int maleToAge, List<Person> peopleList) {
-        return Collections.emptyList();
+        return peopleList.stream()
+                .filter(person -> ((person.getSex().equals(Person.Sex.MAN)
```

Comment:
```suggestion
                .filter(person -> ((person.getSex() == Person.Sex.MAN
```
Let’s  compare enums in this way